src/mono/wasm/runtime/es6/dotnet.es6.pre.js

```diff
@@ -4,3 +4,4 @@ _nativeModuleLoaded = true;
 Module["getMemory"] = function () { return wasmMemory; }
 createDotnetRuntime = Module = createDotnetRuntime(Module);
```

Comment:
Please also define it before createDotnetRuntime so it's available during initialization

Reply:
Before *and* after, like `getMemory`?

Reply:
Let's not do it, unless we know about specific scenarios when this could happen.

Reply:
I'm fine with Before Only if that's what you prefer

Reply:
thanks